example.md

```diff
@@ -0,0 +1,68 @@
+## Setup Guide to Deploy an xERC20 Token
```

Comment:
A part of this is written in the README. Can we merge it with that and consolidate in there so it shows when you open the repo?

Also I think it's a good idea to move the section now to the top of the README as it's in the bottom now!